### ISSUE TITLE
`mackup show <some app>` functionality

### ISSUE THREAD

**Issue by mwilliammyers** [lra/mackup#635]
What do you think about a simple action that would print out the `.cfg` (the local, customized one if found, else the normal one) file for the requested app?

i.e: 
`mackup show git` would print out:

```
[application]
name = Git

[configuration_files]
.gitconfig

[xdg_configuration_files]
git/config
```

This would allow you to see what files _would be_ synced before syncing them.

**Comment by mkwmms** [comment]
Maybe this would be a part of #603?

**Comment by lra** [comment]
Sure, why not.
It does not need tot be part of #603

**Comment by lra** [comment]
Fixed by #1402
